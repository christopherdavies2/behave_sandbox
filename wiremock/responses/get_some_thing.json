{
  "request": {
    "method": "GET",
    "urlPath": "/some/thing"
  },
  "response": {
    "status": 200,
    "jsonBody": {
      "aBool": true,
      "aString": "Hello world!",
      "anInt": 5
    },
    "headers": {
      "Content-Type": "application/json"
    }
  }
}