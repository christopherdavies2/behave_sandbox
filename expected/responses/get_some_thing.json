{
  "aBool": true,
  "aString": "Hello world!",
  "anInt": 5
}